Update SD document (#186)
insightmind/MORR: documents/sd/chapter/Classes/Modules/Keyboard/Keyboard.tex

    \clsdsp{A keyboard user interaction.}

    \begin{attributes}
        \attribute{public System.Windows.Input.Key PressedKey\_System\_Windows\_Input\_Key}{The key that was pressed.}
        \attribute{public string PressedKeyName}{The name of the key on the keyboard that is pressed.}
        \attribute{public System.Windows.Input.ModifierKeys ModifierKeys\_System\_Windows\_Input\_ModifierKeys}{The modifier keys to the key pressed.}

>         \attribute{public string ModifierKeysName}{The name of the key on the keyboard that is pressed.}

Reviewer: ```suggestion
        \attribute{public string ModifierKeysName}{The names of the modifierkeys on the keyboard that are pressed.}
```

>         \attribute{public char MappedCharacter\_Unicode}{The actual user input according to Input locale.}

Reviewer: ```suggestion
        \attribute{public char MappedCharacter\_Unicode}{The actual user input according to input locale.}
```
